client/src/redux/slices/language.slice.js

```diff
@@ -83,6 +83,77 @@ const languageSlice = createSlice({
       )
       state.lessonData.vocab[vocabIndex] = payload.vocab
     },
+    patchSelectedLesson: (state, { payload }) => {
+      /* Patches the fields for a selected lesson. This is called after submitting on <UpdateLesson/>. */
+
+      // Update the data in allLessons
+      const lessonIndex = state.allLessons.findIndex(
+        (element) => element._id === state.currentLessonId,
+      )
+
+      for (var key in payload.lesson) {
+        if (
+          key in state.allLessons[lessonIndex]
```

Comment:
These ones should be fixed with changing the variable declaration from `var` to `let` or `const`.